### PR TITLE
Change CI to Github Actions

### PR DESCRIPTION
**Pull request by kskumgk63**
I have addressed https://github.com/elimity-com/scim/issues/97.

I referred to the following settings.

- golangcli-lint: https://github.com/golangci/golangci-lint-action#how-to-use
- test: https://github.com/mvdan/github-actions-golang#github-actions-for-go

Please review it.

### CODE REVIEW

**Comment by kskumgk63** [comment]
Hi, @di-wu 

I removed `.circleci` but CircleCI test failed even though `yml` does not exist.
Screen Shot 2020-10-27 at 21 49 14

And `.github/workflows` seems be recognized as CircleCi.
Screen Shot 2020-10-27 at 21 48 56


Do you have any ideas?

File: .github/workflows/golangci-lint.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.29
```
Comment (di-wu) [inline .github/workflows/golangci-lint.yml:12]: We should use version `v1.32` since this is the [latest](https://github.com/golangci/golangci-lint/tags).

**Comment by di-wu** [comment]
I will disable CircleCI, you can ignore it.

File: .github/workflows/golangci-lint.yml (hunk 1 above)
Comment (kskumgk63) [reply .github/workflows/golangci-lint.yml]: fix it. https://github.com/elimity-com/scim/pull/98/commits/dbd546b68cefe49e8161fd63f7de369cd7220c7f

**Review by di-wu — APPROVED** [review]

**Comment by di-wu** [comment]
Closes #97

**Pull request merged**